_slides/deepfried_dd.md

```diff
@@ -30,7 +29,42 @@ title: Deepfried_DD
 
 <horizontal/>
 
-## What our dd will look like
+## Applications of `dd`
```

Comment:
Tbh, would move this before the command line parameters

Reply:
I'd say here the problem is that students can't really engage meaningfully in a discussion of application until they know what the utility _does_. I feel that the params give a good description of dd usage and possibilities before we abstract the conversation to uses.

Reply:
Fair enough